.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,26 @@
+**Summary of changes**
+
+<!-- This message body should clearly illustrate what problems it solves. -->
+
+Closes # <!-- It's really useful if your PR references an issue where it is discussed ahead of time. If this PR closes an issue, please list it here. If there are any related issues that this does not close, list them before the "Closes" prefix. -->
+
+### Please don't delete this checklist! Before submitting the PR, please make sure you do the following:
+
+- [ ] I added doc comments to any new public exports, and inline comments to any hard-to-understand areas
+- [ ] My changes generate no new console errors locally
+- [ ] If applicable, include a test that fails without this PR but passes with it
+
+### Tests
+
+- [ ] Run the tests with `pnpm test` and lint the project with `pnpm lint`
```

Comment:
tbh I think we can get rid of this part because the pipeline will yell at them about this anyway

```suggestion
```